honeycomb-core/src/geometry/dim3/tests.rs

```diff
@@ -0,0 +1,220 @@
+// ------ IMPORTS
+
+use super::super::{CoordsFloat, Vector3, Vertex3};
+
+// ------ CONTENT
+
+// --- common
+// scalar equality function
+// ref: https://floating-point-gui.de/errors/comparison/
```

Comment:
I am not totally convinced by the scaling by |A|+|B|.

Can we use a specialized crate like https://docs.rs/float-cmp/latest/float_cmp/ for this kind of stuff?